f2/src/ConfirmationPage.js

```diff
@@ -112,7 +112,8 @@ const submitToServer = async (data, dispatch) => {
   console.log('Submitting data:', data)
   const response = await postData('/submit', data)
   const reportId = await response.text()
-  dispatch({ type: 'saveFormData', data: { reportId, submitted: true } })
+  const submitted = reportId && reportId.startsWith('NCFRS-')
```

Comment:
Do we have to make sure it starts with "NCFRS-"? What if the report id prefix gets changed in the future? Could we instead check to see if report id is not an empty string?

Reply:
If there is an error on the server we return the error message here instead of an id (kinda hacky... though if there's some sort of critical server error this might be all we get).